### ISSUE TITLE
Battleground demolishers still can't be slowed - Strand of the Ancients

### ISSUE THREAD

**Issue by LoveDalaranWow** [dalaranwow/dalaran-wow#5672]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Battleground Demolishers in SotA can't be slowed and are still affected by Crusader Auras
This is a recurring bug since 2015, and basically the only reason the BG is out of the random BG queue.

**Expected behaviour**: Tell us what should happen instead

Veichles should be slowed by any kind of slowing spell and not be affected by paladin auras

**Steps to reproduce the problem**:

1. Enter SotA BG
2. Try to slow any demolisher

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or https://www.wowhead.com/npc=28781/battleground-demolisher
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://www.wowhead.com/npc=28781/battleground-demolisher

**Comment by Golganeth** [comment]
It works as intended, this is the proper behavior in WOTLK retail. If you provide proof that says otherwise then we can take a look.

https://www.mmo-champion.com/threads/1080190-Demolishers-in-Strand-of-The-Ancients

On the side note, crusader aura does not have any kind of effect on the demolishers.
Closing.
